docs/ref/contrib/postgres/fields.txt

```diff
@@ -1007,3 +1007,37 @@ used with a custom range functions that expected boundaries, for example to
 define :class:`~django.contrib.postgres.constraints.ExclusionConstraint`. See
 `the PostgreSQL documentation for the full details <https://www.postgresql.org/
 docs/current/rangetypes.html#RANGETYPES-INCLUSIVITY>`_.
+
+``Serial`` fields
+=================
+
+.. versionadded:: 3.1
+
+.. class:: SerialField(unique=True, editable=False, **options)
+
+    A field similar to :class:`AutoField <django.db.models.AutoField>`
+    except it does not need to be a primary key. It uses a separate postgres
+    sequence to automatically increment the integer value. It stores a 4-byte
+    integer that fits numbers from ``1`` to ``2147483647``.
```

Comment:
While the backing sequence will never use them, zero and negative numbers can be stored when the field is editable. This last sentence seems to suggest otherwise. I have the same concern with the docs for `BigSerialField` and `SmallSerialField`.